recipes/config.rb

```diff
@@ -26,11 +26,3 @@
   variables lazy { node['stackstorm']['config'] }
   action :create
 end
-
-directory 'st2:log_dir' do
```

Comment:
If something breaks, - it's a packaging problem, not the cookbook problem.
Otherwise we'll dig into much more than just `/var/log/st2` dir, which is just small part of that iceberg.

> Example: if someone changes user `st2` -> `st2-random-user` - overriding in a cookbook back to  `st2` won't help, but break everything, since user will be changed in much more places like services https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/st2.st2api.upstart#L10-L11 and entire logic is controlled by deb/rpm. 

Including all checks like that into Integration tests is much better, since we verify entire story, integrity and compatibility there. If something was changed in `deb`/`rpm` logic - that should be caught by upcoming Integration Tests. That's the plan I'd like to stick with.